### DISCUSSION TITLE
紙本與電子

### DISCUSSION THREAD

**Discussion by DarsooK** [1101-GS4719-C/Python-Community-GS4719-C#2917] [General]
大家比較偏好紙本書還是電子書呢？個人閱讀小說、漫畫時偏好紙本；教科書偏好電子。我認為紙本書與電子書各有優缺點，選擇哪一種取決於個人需求與偏好。以下是兩者的比較——紙本書的優點：翻頁的觸感與書香讓閱讀體驗更具儀式感、沒有電子裝置的通知干擾，更容易沉浸於內容；紙本書的缺點：教科書重量較重，攜帶不便、需要實體空間存放，防潮也是問題；電子書的優點：可隨身攜帶數百本書，適合旅行或通勤、快速查找關鍵字或內容，方便學習與研究、許多電子書比紙本書便宜，還有免費書籍資源；電子書的缺點：長時間閱讀可能導致眼睛疲勞、缺乏實體書的情感聯繫感。

**Comment by ed112601513** [comment]
我覺得電子書看久了會很傷眼睛，而紙本書的方便性遠不及電子書，到底要用哪個書好確實是個值得討論的問題，幸好我都不看書，所以這個問題就給大家討論吧。

**Comment by Mminniehuang** [comment]
如果是教科書我無條件支持電子書，減去重量又可以輕鬆的做筆記。至於其他種類，我其實沒有特別的偏好，不過我一直很想買電子書閱讀器，感覺實體書和電子書的優點它都有，攜帶方便又比較不傷眼，雖然還是不能取代實體書翻書的儀式感而且真的好貴...

**Comment by christaaah** [comment]
我喜歡紙本書，不只是閱讀比不傷眼睛，我覺得紙本書對我來說我有想要收藏的價值，電子書的話主要是坊邊，隨時隨地只要有電子設備就能閱讀，但是比較傷眼睛。如果沒有一直移動的話我會選擇閱讀紙本書。

**Comment by Elaine4488** [comment]
就教科書來說的話，大多用電子書。我之前曾經買二手紙本書放在家裡用，電子書做筆記跟帶出去用，我覺得很棒但就是稍嫌麻煩，要找到便宜的二手書。

**Comment by qiu0109** [comment]
我覺得紙本書較適合作為收藏用，電子書不但價格便宜還不用攜帶厚重紙本隨時用手機平板就能觀看，除了看漫畫小說能夠追到最新進度以外，作為教科書還可以減去不小的重量，也方便整理筆記，因此我比較偏好電子書。

**Comment by yue083139** [comment]
我個人看小說漫畫是蠻喜歡紙本書的，在閱讀紙本書的時候，比較能夠讓人沈浸在閱讀的氛圍之中，有時候用電子書就會很容易被訊息打斷干擾，而上課做筆記也習慣要用紙本去做紀錄更能夠記住知識，不過好像有那種單純for閱讀的裝置就是了，會想嘗試看看。

**Comment by kai1660** [comment]
我認為如果是練習題目我比較喜歡使用紙本，因為考試時不會用ipad考試，但是如果一班上課，我毀比較傾向於待平板，因為很多老師的教材都是電子檔，ipad就會比較方便。

**Comment by vu45tp6** [comment]
自從進到大學頻繁使用電子書之後，反而開始有點懷念紙本的感覺。像是來回翻書的那種感覺是在平板上沒辦法感受到的

**Comment by RoyLin1014** [comment]
我跟你一樣，教科書的話偏好電子，平板作筆記真的太方便了，發現剛剛筆記的排版不順眼，可以馬上改，還不用整張筆記重寫，但如果是課外書，偏好紙本，看起來比較有感覺

**Comment by SendaiHatsuki** [comment]
不管是課本還是看小說、漫畫，我都是喜歡用紙本書。紙本書特有的質感和味道是平板模擬不出來的，用紙本看書才會讓我真的有閱讀的感覺。

**Comment by lilipop6405** [comment]
我其實很矛盾。我比較喜歡看紙本的教科書，有翻書的感覺，而且比較好閱讀；可是一要做筆記，電子書就好記很多。但是一定要選一個的話我可能還是會選電子書。不會很重，期中、期末考前就準備一個平板就好，不然紙本書還要帶4、5本厚重的書去k中... 會先累垮在路上。

**Comment by Kaz9487** [comment]
我跟你一樣，小說偏好紙本，有那種翻頁的儀式感，也能夠享受文字的魅力。漫畫的話我比較喜歡電子，畢竟漫畫就小小一本。但教科書真的還是電子版方便，查資料快又省空間。兩種各有優缺，重點是看個人需求。

**Comment by yuuu3957** [comment]
除了教科書外，其實比較喜歡看紙本書的感覺，翻書會有種爽感。
但現實是現在不是住家裡，不太方便將紙本書帶上來，空間也不夠放，因此比較常看電子書，畢竟不占空間，且走到哪裡都能看。

**Comment by yuku-huang** [comment]
紙本書帶來的閱讀儀式感和專注度，是電子書難以取代的，但在攜帶、查找、價格等方面，電子書又更方便。兩者的優缺點很清楚，選擇主要看使用情境和個人習慣。你這樣分開使用，既保留了紙本的情感體驗，也利用了電子書的效率，是種平衡的做法。

**Comment by ryan8967** [comment]
上課我更習慣使用電子。不過我也開始注意到一些專為閱讀設計的電子書裝置，能夠降低藍光影響，感覺是個不錯的選擇。如果有機會的話會想嘗試看看。課外讀物的部分我會想翻紙本

**Comment by yurukung** [comment]
教科書來講的話習慣是電子，因為一個平板就可以帶著走，加上老師常常會把ppt放在eeclass上，下載很方便。但看久了常常會覺得眼壓很高，有點暈暈的。看到某些同學使用的是紙本的方式，有時候也覺得很不錯。如果是課外書的話習慣用紙本，因為平常已經用電子太久了。

**Comment by Pyra-0** [comment]
教科書的部分我是電子派的，畢竟原文書是又厚又重，不會想搬著這幾塊磚頭跑來跑去。在其他地方我都會選紙本，除了我很可憐沒有平板以外，我覺得除了用紙本比在螢幕上寫更有實際的感覺，還能更容易的把一些會干擾的因素排除，像是你的筆記本總不會突然有訊息或突然當機，對吧。

**Comment by Welly940504** [comment]
如果是我自己的話，我會偏好紙本書多一點，但也不是完全排斥電子書，主要是看情境。

紙本書對我來說有種特別的「儀式感」，翻頁的觸感、書的重量、甚至書香味，會讓人更專注在閱讀上，不容易被分心。而且當我在讀比較深的內容或是需要畫重點、貼便條的書，我會很想要用筆在紙上做記號，這是電子書比較難取代的地方。

但電子書也有它超方便的地方，像是出門旅行的時候，不用扛幾本厚書，只要帶一台平板就好；或者臨時要查某個主題，電子書搜尋功能快很多，還可以同步不同裝置，對學習或整理資料來說很實用。

**Comment by 12388833** [comment]
我記得有研究說過，紙本書翻閱頁面的感覺，可以幫助人記憶書中的內容，除此之外電子書的好用程度真的太高了，可以用筆記軟體，讓我們直接在書上做筆記，還可以大大減輕背包的重量，畢竟原文書真的很重。

**Comment by shrshu38** [comment]
我喜歡用 iPad 看電子書，搭配手寫筆記真的很方便。查關鍵字、畫重點都很快，也不用背重書。出門只帶一台平板，過去幾年看的書都在裡面可以隨意翻看。久了也習慣螢幕閱讀，反而更有效率。

**Comment by jaychang1027** [comment]
我喜歡用ipad看電子書，而且很多教授上課的是用簡報上課，所以有ipad就方便很多，而且畫重點查關鍵字甚麼的也都很快，最大的方便就是回家的時候只要帶ipad回家就夠了，所有上課用的簡報一台平板就能搞定，換作是過去紙本的話可能就要帶一堆書回家，占空間又很重

**Comment by ichenx2** [comment]
不論是課外讀物還是教科書，我其實都還是比較喜歡紙本書，總覺得紙本比較有「在讀書」的感覺，翻頁、畫重點都更有沉浸感。但說實話，但最近幾年我讀的還是以電子書居多，主要是因為真的比較方便，而且很多書電子版的價格會比紙本便宜不少，有時候甚至能找到免費資源，身為對學生，免費對我最有吸引力。

**Comment by brianou583** [comment]
我的經驗也跟你滿類似的。讀小說或漫畫時，我偏好紙本書，可能是因為那種「翻頁的節奏」跟「實體的存在感」讓我更容易投入情節。而在讀教科書、技術類資料時，電子書的優勢就非常明顯——尤其是搜尋功能、加註解、同步筆記這些，對理解和整理都很有幫助。

不過我也發現，如果是長時間需要深度理解的內容（例如準備考試），我有時還是會印出來讀，因為那樣我能在紙上隨手畫重點、劃線、做筆記，思考會比較有系統。

**Comment by kyc3318** [comment]
我的經驗是，電子書(紙質)真的方便很多，不會傷眼睛，也可以一次帶很多的書在口袋裡

**Comment by LINSHIHMIN** [comment]
我個人看漫畫或是教科書都習慣用電子，尤其是漫畫用電子書看跨頁的時候真的很舒適⋯⋯紙本可能會被裁到，但電子書可以很平整而且完美的呈現。紙本也喜歡但通常就是買來收藏用的，頂多翻個一兩次內容，有時候拿起來欣賞一下封面這樣。

**Comment by alice20040127** [comment]
如果是為了閱讀的部分的話我會選擇電子的，比較方便攜帶，在自己的電子設備裡面走到哪隨時想看書就能打開來看，不用帶著厚厚的一本書到處走，也不會有突然想看書卻沒帶的問題。如果是為了紀念的話我就會選擇紙本的，就是覺得實體比較有紀念價值，珍藏起來也比較不會有弄丟的問題吧。

**Comment by roy79231** [comment]
對於教科書方面的話我會比較偏好電子書，畢竟好攜帶並且上不同課還不需要帶多本書，至於平時閱讀的話我會比較偏好實體書，長時間使用螢幕去看書的內容其實對眼睛挺傷的，而且我看螢幕白底黑字還會看到頭暈，所以我會比較喜歡讀實體。

**Comment by ghost** [comment]
如果是上課要用的原文書的話，我會比較偏愛電子書，畢竟那些書都蠻厚的，感覺背著好幾本跟字典差不多厚的書上下課不是很好的決定；但如果是平常要看的書的話，我會偏向紙本書，書香的味道還是蠻吸引人的。

**Comment by wuyu06** [comment]
我覺得電子書方便很多，在上面做各種註記塗塗改改都很方便，可以很簡單的就用不同顏色標示，而且在分享筆記時可以直接存PDF檔案

**Comment by Ting940419** [comment]
說到紙本書跟電子書，我覺得真的看情境耶。像我自己，看小說或漫畫的時候會偏好紙本，翻頁的那個手感還有書本的味道，真的有一種很療癒的感覺，特別有「在讀書」的儀式感。而且紙本不會一直跳通知，不會被手機訊息干擾，超適合想靜下來看書的時候用。

但如果是看教科書，我就比較傾向用電子版。一來比較不佔空間，尤其是出門讀書的時候，不用背一本厚重的書很爽；二來是電子書可以直接搜尋關鍵字，找重點超快，筆記功能也很方便。

當然兩種都有缺點啦。像紙本書很佔空間、搬家麻煩，又怕潮濕發霉；電子書的話就是看太久眼睛會酸，而且再怎麼看還是少了那種摸得到書頁的真實感，情感連結沒那麼強

**Comment by kyc3318** [comment]
如果是看書的話，我還是比較偏向紙本

**Comment by wayne935** [comment]
我認同閱讀小說、漫畫時偏好紙本；教科書偏好電子。這樣的觀點，因為教科書真的好重，而且需要在上面做筆記。我不希望我的紙本書上面有很多筆記，這樣看起來會很雜亂。如果是小說的話就不需要在上面做筆記。所以可以用紙本的。而且小說都有一種香味，跟教科書是不一樣的。

**Comment by Piyenparty** [comment]
我個人在享受過電子產品帶來的方便性後就回不去了，大一上的我常常用紙本筆記，但會遇到許多問題、例如容易損毀，墨水到處抹等問題，而且整理相對麻煩，又容易東漏一張西漏一張，遇到些電路圖還要用手繪

ipad帶給我的助益真的很大，他幫我系統性的整理文件，筆記可以做得工整、多彩、各種格式的內容皆可收錄，甚至要貼連結、照片、移動筆記內容、拷貝文件內容等等功能都方便，但偶爾會傳出災情，例如文件丟失等等不可抗力

不過相比兩者，我還是喜歡電子產品，雖然偶爾會反璞歸真用回紙本筆記，但電子的還是NO .1

**Comment by Mminniehuang** [comment]
我個人是比較喜歡電子，而且有些老師上課也都用投影片上課，也不會說課本在第幾頁，大一有試過用紙本，發現根本跟不上，而且還要一直浪費時間找在哪裡，找到後面會直接發呆一節課。
